TeamCode/src/main/java/org/firstinspires/ftc/teamcode/GMR/TeleOp/RR_TeleOp_V1.java

```diff
@@ -0,0 +1,91 @@
+package org.firstinspires.ftc.teamcode.GMR.TeleOp;
+
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import com.qualcomm.robotcore.hardware.DcMotor;
+
+import org.firstinspires.ftc.teamcode.GMR.Robot.Robot;
+
+/**
+ * Created by pston on 11/18/2017
+ */
+
+@TeleOp(name = "Relic Recovery TeleOp V1", group = "test")
+public class RR_TeleOp_V1 extends OpMode {
+
+    private int currentLiftPosition;
+
+    private Robot robot;
+    private double topLeftPosition = 0.37;// Open 0.37, Close: 0
+    private double topRightPosition = 0.26;// Open: 0.26, Close: 0.65
+    private double bottomLeftPosition = 0.55;// Open: 0.55, Close: 0.117
+    private double bottomRightPosition = 0.2;// Open: 0.2, Close: 0.67
```

Comment:
#Fix Please add spaces before the comments, e.g. `private double topLeftPosition = 0.37; // Open 0.37, Close: 0`
#Compliment Nice! I like the fact you documented the open and close positions in comments.